semana17/labenuSystem/src/data/studentList.ts

```diff
@@ -0,0 +1,9 @@
+import { connection } from "../connection";
+
+
+export default async function studentList(student_name: string) {
+    await connection.raw(`
+        INSERT INTO classes (students_list)
+        VALUES ('${student_name}');
+    `)
+}
```

Comment:
Para fazer a adição de estudante de acordo com a sua tabela, você teria que usar uma query UPDATE. Da forma que você fez ele manda um erro por não completar as colunas NOT NULL